examples/resources/awscc_ec2_volume/volume_basic.tf

```diff
@@ -0,0 +1,11 @@
+resource "awscc_ec2_volume" "example" {
+  availability_zone = "us-west-2a"
+  size              = 40
+
+  tags = [
+    {
+      key =  "Name"
+      value = "HelloWorld"
+    }
+  ]
```

Comment:
this tag is not relevant, I suggest to remove it

Reply:
Modified the  tag values